net.kvirc.KVIrc.yaml

```
@@ -0,0 +1,24 @@
+app-id: net.kvirc.KVIrc
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+rename-appdata-file: net.kvirc.KVIrc5.appdata.xml
+rename-desktop-file: net.kvirc.KVIrc5.desktop
+rename-icon: kvirc
+command: kvirc
+
+finish-args:
+  - --share=network
+  - --socket=x11
```

```suggestion
  - --socket=x11
  - --share=ipc
```
this is required for X11.

Also, no wayland support? (since Qt has)

---

I can't test the Wayland support in my setup. I will think a solution as VM